.github/workflows/pull_request.yml

```
@@ -7,14 +7,14 @@ jobs:
     name: Test pull requests
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
-      with:
-        fetch-depth: 1
-    - name: Use Node.js 10.x
-      uses: actions/setup-node@v1
-      with:
-        node-version: 10.x
-    - name: npm install, build, and test
-      run: |
-        npm ci
-        npm test
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+      - name: Use Node.js 10.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10.x
+      - name: npm install, build, and test
+        run: |
+          npm ci
+          npm test
```

Maybe for another PR, but generally I've put `npm ci` and `npm test` in different run steps, but if you want to do it in one step:

```suggestion
          npm install-ci-test
```